planetiler-core/src/main/java/com/onthegomap/planetiler/archive/TileArchiveMetadata.java

```diff
@@ -175,6 +177,14 @@ public TileArchiveMetadata withJson(TileArchiveMetadataJson json) {
    * https://github.com/FasterXML/jackson-databind/issues/3439
    */
 
+  private static Map<String,String> mergeMaps(Map<String,String> m1, Map<String,String> m2) {
+    return Stream.concat(m1.entrySet().stream(), m2.entrySet().stream())
+      .collect(Collectors.toMap(
+        Map.Entry::getKey,
+        Map.Entry::getValue,
+        (value1, value2) -> value2));
+  }
+
```

Comment:
Might be slightly more straightforward to use putAll here, that makes it explicit which direction we are clobbering too:

```java
var result = new TreeMap<>(m1);
result.putAll(m2);
return result;
```